tests/lodash-extras-tests.js

```diff
@@ -0,0 +1,256 @@
+      _.runInContext().merge(_, all);
+    });
+
+    describe('isPresent', () => {
+      it('exists', () => expect(!_.isUndefined(_.isPresent) && !_.isNull(_.isPresent)).to.be.true);
+
+      it('Identifies present objects', () => {
+        const testObject = { };
+        expect(_.isPresent(testObject)).to.be.true;
+      });
+
+      it('Identifies undefined objects', () => {
+        let testObject;
+        expect(_.isPresent(testObject)).to.be.false
+      });
+
+      it('Identifies null objects', () => {
+        const testObject = null;
+        expect(_.isPresent(testObject)).to.be.false
+      });
+    });
+
+    describe('isBlank', () => {
+      it('exists', () => expect(_.isPresent(_.isBlank)).to.be.true);
+
+      it('Identifies blank objects', () => {
+        let testObject;
+        expect(_.isBlank(testObject)).to.be.true
+      });
+
+      it('Identifies non-blank objects', () => {
+        const testObject = { };
+        expect(_.isBlank(testObject)).to.be.false;
+      });
+    });
+
+    describe('isDate', () => {
+      it('exists', () => expect(_.isPresent(_.isDate)).to.be.true);
+
+      it('Identifies dates', () => {
+        const testObject = new Date();
+        expect(_.isDate(testObject)).to.be.true
+      });
+
+      it('Identifies non-dates', () => {
+        const testObject = { };
+        expect(_.isDate(testObject)).to.be.false;
+      });
+    });
+
+    describe('isPromise', () => {
+      it('exists', () => expect(_.isPresent(_.isPromise)).to.be.true);
+
+      it('Identifies promises', () => {
+        const testObject = new Promise((reslove, reject) => resolve());
+        expect(_.isPromise(testObject)).to.be.true
+      });
+
+      it('Identifies non-promises', () => {
+        const testObject = { };
+        expect(_.isPromise(testObject)).to.be.false;
+      });
+    });
+
+    describe('is', () => {
+      beforeEach(function() {
+        sinon.spy(console, 'warn')
+        sinon.spy(console, 'error');
+      });
+
+      afterEach(function() {
+        console.warn.restore();
+        console.error.restore();
+      });
+
+      it('exists', () => expect(_.isPresent(_.is)).to.be.true);
+
+      it('Identifies match', () => {
+        const testObject = { };
+        const conditions = ['isPresent', 'isObject'];
+        expect(_.is(testObject, conditions)).to.be.true
+      });
+
+      it('Identifies non-match', () => {
+        const testObject = { };
+        const conditions = ['isPresent', 'isBlank'];
+        expect(_.is(testObject, conditions)).to.be.false
+      });
+
+      it('Displays an error for single condition but completes', () => {
+        const testObject = { };
+        const conditions = ['isPresent'];
+        expect(_.is(testObject, conditions)).to.be.true
+        expect(console.error.calledOnce).to.be.true;
+        expect(console.error.getCall(0).args[0]).to.equal("Don't call `is` helper with just one condition- use that condition directly");
```

Comment:
We can but I wanted to use the exact message so that we know it matched the logic we expected. That way we don't accidentally pass a tests that is matching the wrong condition but still returning false.